src/Plugin.php

```diff
@@ -51,10 +50,6 @@ public function bootstrap(PluginApplicationInterface $app): void
             ));
             EventManager::instance()->on(new Event\ProfilerListener());
         }
-
-        if (Configure::read('debug')) {
-            Configure::write(sprintf('%s.cache', TwigView::ENV_CONFIG), false);
```

Comment:
This change was recently merged into `master` and there hasn't been a release yet, so best to make similar change on master too.